feubot.py

```diff
@@ -58,6 +58,58 @@ async def on_ready():
         print('------')
         await bot.change_presence(activity=discord.Game(name="Reading the doc!"))
 
+    @bot.event
+    async def on_raw_reaction_add(payload):
+        messageID = payload.message_id
+        try:
+            with open("BCRoleReaction.txt", "r") as file:
+                reactMessage = int(file.read())
+
+        except (FileNotFoundError, IOError):
+            return
+
+        if messageID == reactMessage:
+            guild_id = payload.guild_id
+            guild = discord.utils.find(lambda g : g.id == guild_id, bot.guilds)
+
+            if payload.emoji.name == "🐱":
```

Comment:
It's a bit sad to be hardcoding this (and also needing to re-deploy every time we want to add to it), but I don't have a better idea at the moment.

Reply:
That's true. I hardcoded it this time since we don't really need a robust system yet.
If it comes to that, updating the script to take a role ID or name and a corresponding emote to add reaction roles is probably the way to go.

Reply:
The overhaul has been done and now allows for multiple messages to have reaction roles that can be customized via bot commands.

`!setReactionRole <messageID>, <role>, <reaction>` (Admin only): Sets a reaction role on a specified message by ID
`unsetReactionRole <messageID>, <reaction>` (Admin only): Removes a reaction role from a specified message by reaction. using `ALL` as the reaction argument will remove all role reactions from the message
`!listReactionRoles`: Lists all reaction roles currently set across all messages. Displays a warning when the role doesn't exist.